src/expr/src/sig/func.rs

```diff
@@ -167,6 +171,9 @@ mod tests {
         // handle them specially without relying on FuncSigMap.
         let expected = expect_test::expect![[r#"
             {
+                ToTimestamp1: [
+                    "to_timestamp1(varchar, varchar) -> timestamptz/timestamp",
+                ],
```

Comment:
We can actually exclude deprecated signatures from `new_map` above so that `ToTimestamp1`/`ArrayLength`/`Cardinality` are no longer considered duplicates.